resources/search.py

```diff
@@ -38,20 +40,17 @@ def post(self):
                     schema:
                       type: array
                       items:
-                        $ref: '#/components/schemas/ObjectBase'
+                        $ref: '#/components/schemas/ObjectListItemResponse'
             400:
                 description: When request body or query syntax is invalid
         """
-        schema = SearchSchema()
-
+        schema = SearchRequestSchema()
         obj = schema.loads(request.get_data(as_text=True))
 
         if obj.errors:
             return {"errors": obj.errors}, 400
 
         query = obj.data["query"]
-        logger.info('search', extra={'query': query})
```

Comment:
We don't want to log this anymore?

Reply:
Nope, this endpoint is deprecated anyway